rpc/documentation/api.md

```diff
@@ -1476,3 +1477,86 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+## `TicketBuyerLoaderService`
```

Comment:
This can be just TicketBuyerService and the other ticket buyer RPCs can be added to it later.